website/src/components/pages/Homepage/Homepage.js

```diff
@@ -73,7 +73,7 @@ const Homepage = () =>{
         <div class="row featurette">
             <div class="col-md-7">
             <h2 class="featurette-heading">Projekt Apollon. <span class="text-muted">Woher kommt der Name?</span></h2>
-            <p class="lead">Apollon ist ein griechisch-römischer Gott, geboren auf der Insel Delos: einer heiligen Stätte. Im selben Stile wurde dieses Projekt gegründet, aufbauend auf unserer heiligen Stätte des Zusammenhalts, der Teamarbeit und dem höchsten Grade an Motivation, das bestmöglichste Ergebnisse für unsere Kunden zu liefern. Als Beschützer der Künste ist Apollon ein Mahnmal unseres Teams, selbst Produkte zu entwickeln, die als "Kunst" gesehen werden.</p>
+            <p class="lead">Apollon ist ein griechisch-römischer Gott, geboren auf der Insel Delos: einer heiligen Stätte. Im selben Stile wurde dieses Projekt gegründet, aufbauend auf unserer heiligen Stätte des Zusammenhalts, der Teamarbeit und dem höchsten Grade an Motivation, das bestmöglichste Ergebnis für unsere Kunden zu liefern. Als Beschützer der Künste ist Apollon ein Mahnmal unseres Teams, selbst Produkte zu entwickeln, die als "Kunst" gesehen werden.</p>
```

Comment:
"Das bestögliche Ergebnis" oder "die bestmöglichen Ergebnisse"

Reply:
Mir egal. Find beides passend. Finde in der Einzahl klingts halt irgendwie absoluter.

Reply:
> Da das Adjektiv bestmöglich mit »best« bereits einen Superlativ enthält, kann es nicht nochmals gesteigert werden: »Das ist die bestmögliche (nicht: bestmöglichste) Lösung.« Das gilt auch für die Substantivierung: »Ich werde mein Bestmögliches (nicht: Bestmöglichstes) tun.«

Ist halt grammatikalisch nicht korrekt...

Reply:
Oh, das stimmt natürlich. Änder ich gleich. Trotzdem, Einzahl oder Mehrzahl?

Reply:
Das ist mir relativ egal.